files/chaincode/src/map/map.go

```diff
@@ -44,24 +45,29 @@ type SimpleChaincode struct {
 // Init is a no-op
 func (t *SimpleChaincode) Init(stub shim.ChaincodeStubInterface) pb.Response {
 	creator, err := stub.GetCreator();
-	fmt.Printf("Init creator: %s", string(creator[:]))
+	creatorString := url.PathEscape(string(creator[:]))
```

Comment:
By the way, I wonder if  for some imaginary reasons, stub.GetCreator returns err, so creator might be bogus or nil, would all of `url.PathEscape(string(creator[:]))` handle that situation correctly?
Probably it worth to move error check earlier?

Reply:
I totally agree. Fixed